src/modules/ie-objects/ObjectDetailPage.tsx

```diff
@@ -479,9 +480,9 @@ export const ObjectDetailPage: FC<DefaultSeoInfo> = ({ title, description, image
 		mediaInfo?.licenses?.includes(IeObjectLicense.BEZOEKERTOOL_CONTENT) &&
 		visitorSpace?.status === VisitorSpaceStatus.Active &&
 		!isKiosk;
-	const canDownloadMetadata: boolean =
-		(useHasAnyPermission(Permission.EXPORT_OBJECT, Permission.DOWNLOAD_OBJECT) &&
-			(hasAccessToVisitorSpaceOfObject || isPublicNewspaper)) ||
+	const canDownloadMetadata: boolean = useHasAnyPermission(Permission.EXPORT_OBJECT) || false;
```

Comment:
this function defaults to false, so you don't need to specify it
```suggestion
	const canDownloadMetadata: boolean = useHasAnyPermission(Permission.EXPORT_OBJECT);
```